apollo-core/src/main/java/com/spotify/apollo/core/ServiceImpl.java

```diff
@@ -515,33 +479,7 @@ public Service build() {
     }
   }
 
-  static class ExecutorServiceCloseable implements Closeable {
```

Comment:
Specifically for closing executors, you mean? If so, would something like
```
  environment.closer().register(ExecutorCloser.of(executor));
```
work?